src/server/API/appsettings.json

```
@@ -0,0 +1,53 @@
+{
+  "AllowedHosts": "*",
+  "CacheSettings": {
+    "SlidingExpiration": 2
+  },
+  "JwtSettings": {
+    "key": "S0M3RAN0MS3CR3T!1!MAG1C!1!",
+    "tokenExpirationInMinutes": 60,
+    "refreshTokenExpirationInDays": 7
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information",
+      "Hangfire": "Warning"
+    }
+  },
+  "MailSettings": {
+    "From": "info@codewithmukesh.com",
+    "Host": "smtp.ethereal.email",
+    "Port": 587,
+    "UserName": "adaline.pfannerstill49@ethereal.email",
+    "Password": "vAKmWQB8CyPUBg8rBQ",
+    "DisplayName": "Mukesh Murugan",
```

```suggestion
    "From": "boilerplate@accentdesign.co.uk",
    "Host": "smtp.ethereal.email",
    "Port": 587,
    "UserName": "jeffry27@ethereal.email",
    "Password": "9U2sJHZsxd8ekZF383",
    "DisplayName": "Accent Boilerplate",
```

I setup an account as part of testing if you want to use it :) ^^